README.md

```diff
@@ -1,27 +1,20 @@
-# Core Sitemaps #
-**Contributors:** joemcgill, pacifika, kburgoine  
-**Tags:** seo, sitemaps  
-**Requires at least:** 5.2.0  
-**Tested up to:** 5.2.3  
-**Stable tag:** 0.1.0  
-**License:** GPLv2 or later  
-**License URI:** https://www.gnu.org/licenses/gpl-2.0.html
-
-A feature plugin to integrate basic XML Sitemaps in WordPress Core
-
-## Description ##
-
-See: https://make.wordpress.org/core/2019/06/12/xml-sitemaps-feature-project-proposal/
-
-
-## Documentation ##
-
-- Local Setup: [Local Setup Documentation Section](/docs/SETUP.md/).
-- Contributing: [Contributing Documentation Section](/docs/CONTRIBUTING.md)
-- Testing: [Testing Documentation Section](/docs/TESTING.md).
-
-
-## Changelog ##
-
-### 0.1.0 ###
-* In progress...
+# Core Sitemaps #
+**Contributors:** joemcgill,  
```

Comment:
Looks like the readme task stripped the list of contributors.

Reply:
Oh, looks like they were added here manually before. Will add them back properly.